packages/client/src/__tests__/batching.test.ts

```diff
@@ -1,21 +1,23 @@
 import { blog } from '../fixtures/blog'
 import { getDMMF } from '../generation/getDMMF'
 import { DMMFClass, makeDocument } from '../runtime'
-import { PrismaClientFetcher } from '../runtime/PrismaClientFetcher'
+import { RequestHandler } from '../runtime/RequestHandler'
 
 describe('batching', () => {
   test('basic batching', async () => {
     const dmmf = new DMMFClass(await getDMMF({ datamodel: blog }))
     const batches: any[] = []
     const requests: any[] = []
 
-    const fetcher = new PrismaClientFetcher({
+    const fetcher = new RequestHandler({
       $connect: () => Promise.resolve(),
       _engine: {
+        // @ts-expect-error
```

Comment:
```suggestion
        // @ts-expect-error
```
Why do we need this now? (and in other places) 👀

Reply:
`RequestHandler` is now better typed and inherits from the poor internal types that we have for `Client`. That is enough though, as `_engine` is not public API and used to resolve to `any`.